cypress/integration/bookings/list.js

```diff
@@ -1,53 +1,72 @@
-import { booking1, booking2, booking2Details} from '../../lib/helper.js'
+import { booking1Details, bookingsList } from '../../lib/helper.js'
 
 describe('List bookings', () => {
-  before(async () => {
-    await cy.exec('dotenv -e .env.test -- npx prisma migrate reset --force')
-  })
-
   it('Should not have permissions when not logged in', () => {
     cy.visit('http://localhost:3000/booking/list')
     
     cy.contains('Welcome to SalesWell')
-    cy.contains(booking1).should('not.exist')
-    cy.contains(booking2).should('not.exist')
-    cy.contains('No bookings!').should('not.exist')
+    cy.contains(bookingsList[0]).should('not.exist')
+    cy.contains(bookingsList[1]).should('not.exist')
   })
 
   it('Should have permissions when logged in as user', () => {
     cy.login('normal@example.com', 'password')
     cy.visit('http://localhost:3000/booking/list')
     
-    cy.contains(booking1)
-    cy.contains(booking2)
+    cy.contains(bookingsList[0])
+    cy.contains(bookingsList[1])
+    cy.contains(bookingsList[17])
```

Comment:
let’s add a few comments here explaining why 0, 1 and 17 should exist and 18 not.
Also in order to avoid repetitive code, I would extract this to a function bookingsInListExist and I would call it for both tests (user and admin)